lib/melodica_inventory_web/controllers/admin/item_controller.ex

```diff
@@ -42,4 +54,10 @@ defmodule MelodicaInventoryWeb.Admin.ItemController do
         |> redirect(to: category_path(conn, :show, item.variation.category_id))
     end
   end
+
+  defp update_item(item, item_params) do
+    Multi.new()
+    |> Multi.update(:item, Item.changeset(item, item_params))
+    |> Multi.run(:image, fn state -> ImageOperations.upload_images(state, item_params["image"]) end)
```

Comment:
I think the `upload_images` should be outside of the DB transaction here. This is because it does API calls to the `Cloudex` service and if the API call fails we want to still keep the previously uploaded image records in the DB. If the upload is in the transaction, it is possible that an image will be uploaded in the cloud and there is no DB records for it, because the transaction rolled back and removed the record.